test/powershell/Language/Parser/UsingModule.Tests.ps1

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+Describe "Using module" -Tag "CI" {
```

Comment:
We have scripting.Classes.using.tests.ps1 for such tests. Please move the tests to the file.